### PR TITLE
Fyst 1158 add grocery tax credit to id review

### PR DESCRIPTION
**Pull request by squanto**
## Link to pivotal/JIRA issue
- https://codeforamerica.atlassian.net/browse/FYST-1158

## Is PM acceptance required? (delete one)
- Yes - don't merge until JIRA issue is accepted!

**Reminder**: merge main into this branch and get green tests before merging to main
## What was done?
- Add grocery credit to final idaho review page
 
## How to test?
- go through idaho state file process
- verify final review page

### CODE REVIEW

File: app/views/state_file/questions/id_review/edit.html.erb [hunk 1]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (arinchoi03) [inline app/views/state_file/questions/id_review/edit.html.erb:1]: remove line

File: app/views/state_file/questions/id_review/edit.html.erb [hunk 2]
```diff
@@ -36,3 +43,4 @@
 
   <%= render "state_file/questions/shared/review_footer" %>
 <% end %>
+
```
Comment (arinchoi03) [inline app/views/state_file/questions/id_review/edit.html.erb:46]: remove line?

File: app/views/state_file/questions/id_review/edit.html.erb [hunk 3]
```diff
@@ -20,6 +21,12 @@
         <p><%=number_to_currency(current_intake.health_insurance_paid_amount || 0, precision: 2)%></p>
         <%= link_to t("general.edit"), StateFile::Questions::IdHealthInsurancePremiumController.to_path_helper(return_to_review: "y"), class: "button--small" %>
       </div>
+
+      <div class="spacing-below-5">
+        Idaho Grocery Credit
+        <%= current_intake.calculator.grocery_credit_amount %>
+        <%= link_to t("general.review_and_edit"), StateFile::Questions::IdGroceryCreditReviewController.to_path_helper(return_to_review: "y"), class: "button--small" %>
```
Comment (arinchoi03) [inline app/views/state_file/questions/id_review/edit.html.erb:28]: it looks like we use `edit` for the other blocks here and "review and edit" for just this grocery credit -- looking at the figma it looks like we want "Review and Edit" for all the other buttons on this page, could we confirm with product?

**Comment by arinchoi03** [comment]
you'll need to `annotate` as well to fix the ruby test failures

File: app/views/state_file/questions/id_review/edit.html.erb (hunk 3 above)
Comment (squanto) [reply app/views/state_file/questions/id_review/edit.html.erb]: just double checked with product, and it should be Edit everywhere. Updated 👍

**Review by anisharamnani — APPROVED** [review]
I read the comments & the errors seem to be resolved. Thanks @squanto!

**Pull request merged**